README.md

```diff
@@ -78,6 +78,24 @@ assert.strictEqual(result, 4);
 
 We are [also open to adding an array overload](https://github.com/unsplash/pipe-ts/issues/5), so any number of funtions can be passed.
 
+## When `this` matters
+
+If a function passed to `pipe` or `pipeWith` relies on a specific context of execution ([`this`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/this)), you cannot call it by passing the reference (_point-free style_).
+
+Use an anonymous function instead.
+
+**Bad**
+
+```ts
+pipeWith('foo', JSON.parse);
+```
+
+**Good**
+
+```ts
+pipeWith('foo', input => JSON.parse(input));
```

Comment:
Or `JSON.parse.bind(JSON)`?

Reply:
Correct!

(I changed the example to `localStorage` instead of `JSON` since JSON seems to be working with any `this` after all.)